tutor/env.py

```diff
@@ -64,6 +64,7 @@ def _prepare_environment() -> None:
             ("HOST_USER_ID", utils.get_user_id()),
             ("TUTOR_APP", __app__.replace("-", "_")),
             ("TUTOR_VERSION", __version__),
+            ("IS_NEXT", tutor.__about__.__app__.__version_suffix__ == "next"),
```

Comment:
I have an issue with the "IS_NEXT" name, which is not very explicit. Maybe "IS_ON_NEXT_BRANCH"? Or "IS_NEXT_VERSION"?

Also, we can simplify this line to `__app__.__version_suffix__ == "next"`.

Reply:
@regisb How about just TUTOR_NEXT ? It's not very explicit either, but since "Tutor Next" will be an established term in the docs (like "Tutor Nightly" is now), I think that a user looking at a boolean variable named TUTOR_NEXT will be able to deduce the meaning.

Reply:
That would translate to `IS_MAIN` or `TUTOR_MAIN` with the [latest decision](https://discuss.openedx.org/t/tep-rename-nightly-next-and-master-release/12994/9). Between these two propositions I prefer the original one, `IS_MAIN`.

Reply:
@kdmccormick Are you taking care of changes as per latest decision?

Reply:
@DawoudSheraz Yes

Reply:
I like that the other two Tutor "meta" template variables start with `TUTOR_`, so I was interested in finding a name that had the same prefix. I went with `TUTOR_BRANCH_IS_MAIN`. @regisb @DawoudSheraz if you don't like that, feel free to switch it to any of the other suggestions that we have discussed above--I have no objections to any of them.

Reply:
I like `TUTOR_BRANCH_IS_MAIN`, it's very explicit.